src/components/Header/EmailButton.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from '@components/ui/popover';
+import email from '@public/header__email.svg';
+import fakeAvatar from '@public/header__fakeAvatar.svg';
```

Comment:
Next.js 雖然在local端可以用`import “image” from “public ”`，但是deploy至伺服器後，檔案位置並不相同，會造成部署失敗。

可以直接使用下面的語法，指定公開的路經

- ref:
  - https://vercel.com/north9hex/chillka-frontend/3MCUBWmNQwmF6yAvR8TCCriUHLXu
  - https://stackoverflow.com/questions/76990550/why-am-i-unable-to-import-any-of-the-images-which-are-present-in-the-public-fold

```suggestion
const fakeAvatar = '/header__fakeAvatar.svg';
```